src-plugins/manualRegistration/manualRegistration.h

```diff
@@ -92,6 +91,10 @@ class MANUALREGISTRATIONPLUGIN_EXPORT manualRegistration : public itkProcessRegi
     */
     virtual bool writeTransform(const QString& file);
 
+    void setParameter(int data, int channel);
```

Comment:
That discussion was about ```setInput``` if I remember, and that is because for some strange reason ```medAbstractProcess``` hides all the overloads of ```setInput``` to use only the channel version (which we should change).